src/components/molecules/SettingsLeftNavigation/SettingsLeftNavigation.styled.tsx

```diff
@@ -21,6 +21,5 @@ export const StyledNavigationContainer = styled.div`
   flex-direction: column;
   gap: 16px;
 
```

Comment:
Is that expected? Have you tested it for all pages on both Cloud and OSS?

Reply:
I just changed `padding-right: 20px` to `gap: 20px` as for me it seems more consistant. Did not affect anything.